tl/appendices/3-0-migration-guide.rst

```diff
@@ -13,7 +13,7 @@ Mga Kinakailangan
 - Ang CakePHP 3.x ay nangangailangan ng mbstring na ekstensiyon.
 - Ang CakePHP 3.x ay nangangailangan ng intl na ekstensiyon.
 
-.. babala::
+.. warning::
```

Comment:
i think it fits more on babala

Reply:
`.. note::` is a directive for sphinx and should not be translated. `.. warning::` is also a directive that should not be translated.